app/javascript/flagUserModal/flagUserModal.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { h } from 'preact';
```

Comment:
Yeah these tests are great, but it would be good to have one end-to-end flow for this so that we're also testing the full flow of the modal being opened and initialized properly. It could just be a single "happy path" Cypress test

Reply:
Yeah, thanks for the tip!!!

Reply:
I need to ask. How do I run cypress inside the app docker container?

Reply:
I got this error:

![image](https://user-images.githubusercontent.com/23138717/172175955-d99a968a-408b-410f-8a88-6ff1e01aa16d.png)

Reply:
@aitchiss Does exist some setup doc updated for cypress? Because I am following the current doc and I am got some errors trying to run cypress (with docker or machine).

Reply:
@joaoGabriel55 I'm sorry you're experiencing this. I know in the past other contributors have had trouble running Cypress in Docker. Thanks for weighing in @aitchiss.

Reply:
Hey @joaoGabriel55 unfortunately I don't have experience of running Cypress in the docker environment. Outside of docker you should be able to run `yarn e2e`, select "yes" to the database setup, and the interactive test runner will pop up.

Cypress has [some guidance around running it in docker](https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/), but if it's proving too much of a headache to get running, just let me know and I can collaborate with you on this PR and add that Cypress spec (I don't want your PR getting held up too much just for one spec 😅 )

Reply:
Yes, could be @aitchiss. Thanks a lot